src/js/angular/repositories/controllers.js

```diff
@@ -153,11 +153,13 @@ function LocationsAndRepositoriesCtrl($scope, $modal, toastr, $repositories, Mod
     };
 
     //Get locations
-    $repositories.getLocations()
-        .then((locations) => {
-            $scope.locations = locations;
-        })
-        .finally(() => $scope.loader = false);
+    function getLocations() {
+        $repositories.getLocations()
```

Comment:
Eventually return the promise for consistency.

Reply:
good point